esphome/components/display/__init__.py

```diff
@@ -86,6 +88,10 @@ def validate_rotation(value):
 async def setup_display_core_(var, config):
     if CONF_ROTATION in config:
         cg.add(var.set_rotation(DISPLAY_ROTATIONS[config[CONF_ROTATION]]))
+
+    if CONF_AUTO_CLEAR_ENABLED in config:
+        cg.add(var.set_auto_clear(config[CONF_AUTO_CLEAR_ENABLED]))
+
```

Comment:
So that this can be done unconditionally:
```suggestion
    cg.add(var.set_auto_clear(config[CONF_AUTO_CLEAR_ENABLED]))
```